ui/component/livestreamSuperchats/index.js

```diff
@@ -0,0 +1,4 @@
+import { connect } from 'react-redux';
+import LivestreamSuperchats from './view';
+
+export default connect()(LivestreamSuperchats);
```

Comment:
No need to call connect if there's nothing.  It appears in the profiler, so there is some cost.  We can always add it when there is a need.